docs/ism/index.md

```diff
@@ -34,16 +34,18 @@ A policy is a set of rules that describes how an index should be managed. For in
 4. In the **Define policy** section, enter your policy.
 5. Choose **Create**.
 
-After you create a policy, your next step is to attach this policy to an index or indices.
-You can also include the `policy_id` in an index template so when an index is created that matches the index template pattern, the index will have the policy attached to it:
+After you create a policy, your next step is to set up an `ism_template` so when an index is created that matches the index template pattern, the index will have the policy attached to it:
```

Comment:
Can change to

- After you create a policy, your next step is to attach this policy to an index or indices.
- You can set up an `ism_template` in the policy so when an index is created and matches the ISM template pattern, the index will have this policy attached to it:

Reply:
Also can you link to the ISM template section on Policies page from here